### PR TITLE
Added Apps Update Profile Attribute specs to api-apps-update applicat…

### PR DESCRIPTION
**Pull request by anweshdatla-okta**
Added Apps Update Profile Attribute specs to api-apps-update application page
Modified Apps API Collection to include new request (Update Application Profile Attributes) with a sample response

Resolves: OKTA-426644

## Description:
- **What's changed?** 
    1. Added Apps Update Profile Attribute specs to api->apps->update application page
    2. Modified Apps API Collection to include new request (Update Application Profile Attributes) with a sample response
    
image

- **Is this PR related to a Monolith release?**  No

### Resolves:

* [OKTA-426644](https://oktainc.atlassian.net/browse/OKTA-426644)

### Any Manual Testing Done?
I ran this Update Apps API to update app profile attributes in my local and preview env, added a scope in authorization server and able to see the newly added profile attributes in token preview
image

Primary reviewers: @susanharper-okta @larsjohansen-okta @IanHakes-okta 

Other reviewers: @okta/federations

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 1]
```diff
@@ -3384,7 +3384,190 @@ curl -v -X PUT \
...
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "name": "oidc_client",
+  "label": "oauth2 client app 1",
+  "signOnMode": "OPENID_CONNECT",
+  "credentials": {
+    "oauthClient": {
+      "client_id": 0oa1ivcatpQDkMraA1d7,
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:3415]:
```suggestion
      "client_id": ${clientID},
```

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 2]
```diff
@@ -3384,7 +3384,190 @@ curl -v -X PUT \
...
+    },
+    "appLinks": {
+      "oidc_client_link": true
+    }
+  },
+  "features": [],
+  "signOnMode": "OPENID_CONNECT",
+  "credentials": {
+    "userNameTemplate": {
+      "template": "${source.login}",
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:3481]: The $ symbol means that a dev must input a variable - is that what you mean to convey here?

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 3]
```diff
@@ -3384,7 +3384,190 @@ curl -v -X PUT \
...
+        "default_scope": []
+      }
+    }
+  },
+  "profile": {
+    "label": "oauth2 client app 1"
+  },
+  "_links": {
+    "uploadLogo": {
+      "href": "https://dev-adatla-op3.oktapreview.com/api/v1/apps/0oa1ivcatpQDkMraA1d7/logo",
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:3538]:
```suggestion
      "href": "https://example.okta.com/api/v1/apps/0oa1ivcatpQDkMraA1d7/logo",
```

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 4]
```diff
@@ -3384,7 +3384,190 @@ curl -v -X PUT \
...
+      "hints": {
+        "allow": [
+          "POST"
+        ]
+      }
+    },
+    "appLinks": [
+      {
+        "name": "oidc_client_link",
+        "href": "https://dev-adatla-op3.oktapreview.com/home/oidc_client/0oa1ivcatpQDkMraA1d7/aln177a159h7Zf52X0g8",
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:3548]:
```suggestion
        "href": "https://example.okta.com/home/oidc_client/0oa1ivcatpQDkMraA1d7/aln177a159h7Zf52X0g8",
```

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 5]
```diff
@@ -3384,7 +3384,190 @@ curl -v -X PUT \
...
+        "type": "text/html"
+      }
+    ],
+    "groups": {
+      "href": "https://dev-adatla-op3.oktapreview.com/api/v1/apps/0oa1ivcatpQDkMraA1d7/groups"
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:3553]:
```suggestion
      "href": "https://example.okta.com/api/v1/apps/0oa1ivcatpQDkMraA1d7/groups"
```

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 6]
```diff
@@ -3384,7 +3384,190 @@ curl -v -X PUT \
...
+    },
+    "logo": [
+      {
+        "name": "medium",
+        "href": "https://op3static.oktacdn.com/assets/img/logos/default.6770228fb0dab49a1695ef440a5279bb.png",
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:3558]:
```suggestion
        "href": "https://example.okta.com/assets/img/logos/default.6770228fb0dab49a1695ef440a5279bb.png",
```

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 7]
```diff
@@ -3384,7 +3384,190 @@ curl -v -X PUT \
...
+        "type": "image/png"
+      }
+    ],
+    "users": {
+      "href": "https://dev-adatla-op3.oktapreview.com/api/v1/apps/0oa1ivcatpQDkMraA1d7/users"
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:3563]:
```suggestion
      "href": "https://example.okta.com/api/v1/apps/0oa1ivcatpQDkMraA1d7/users"
```

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 8]
```diff
@@ -3384,7 +3384,190 @@ curl -v -X PUT \
...
+    },
+    "deactivate": {
+      "href": "https://dev-adatla-op3.oktapreview.com/api/v1/apps/0oa1ivcatpQDkMraA1d7/lifecycle/deactivate"
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:3566]:
```suggestion
      "href": "https://example.okta.com/api/v1/apps/0oa1ivcatpQDkMraA1d7/lifecycle/deactivate"
```

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md (hunk 2 above)
Comment (anweshdatla-okta) [reply packages/@okta/vuepress-site/docs/reference/api/apps/index.md:3481]: Yes, thats correct. In this instance, it is just showing what the userNameTemplate should be. This is from the sample response of new API.
you can also find this application object sample response here https://developer.okta.com/docs/reference/api/apps/#example

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md (hunk 5 above)
Comment (anweshdatla-okta) [reply packages/@okta/vuepress-site/docs/reference/api/apps/index.md]: @susanharper-okta modified urls to generic example.okta domain.

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md (hunk 6 above)
Comment (anweshdatla-okta) [reply packages/@okta/vuepress-site/docs/reference/api/apps/index.md]: @susanharper-okta modified urls to generic example.okta domain.

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md (hunk 8 above)
Comment (anweshdatla-okta) [reply packages/@okta/vuepress-site/docs/reference/api/apps/index.md]: @susanharper-okta modified urls to generic example.okta domain.

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 9]
```diff
@@ -3384,7 +3384,190 @@ curl -v -X PUT \
...
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "name": "oidc_client",
+  "label": "oauth2 client app 1",
+  "signOnMode": "OPENID_CONNECT",
+  "credentials": {
+    "oauthClient": {
+      "client_id": ${clientID},
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:3415]:
```suggestion
      "client_id": ${clientId},
```

**Review by susanharper-okta — APPROVED** [review]
LGTM

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 10]
```diff
@@ -9,7 +9,7 @@ The Okta Application API provides operations to manage applications and/or assig
 
 ## Get started
 
-Explore the Apps API: [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/be7367401f4926a945f8)
+Explore the Apps API: [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/a1baf2ec75f797e6af36)
```
Comment (larsjohansen-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:12]: image
The header should be `SSWS {{apikey}}` to keep it consistent with the rest of the collection.

Comment (larsjohansen-okta) [reply packages/@okta/vuepress-site/docs/reference/api/apps/index.md:12]: Is this particular to updating the client? Can you not add the profile attributes during creation? If you can during creation, wouldn't it make sense to re-use the existing request under `Add Application` -> `Add OAuth2 Client App`?

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 11]
```diff
@@ -3384,7 +3384,190 @@ curl -v -X PUT \
     }
 }
 ```
+#### Update application level profile attributes
 
+Updates the Application profile attributes
```
Comment (larsjohansen-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:3389]: Again, not sure why this is specific to update

**Review by larsjohansen-okta — CHANGES_REQUESTED** [review]

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md (hunk 10 above)
Comment (anweshdatla-okta) [reply packages/@okta/vuepress-site/docs/reference/api/apps/index.md]: 1. good catch, modified header value to SSWS {{apikey}}
2. As discussed, one cannot add it through UI. After discussing in [jira](https://oktainc.atlassian.net/browse/OKTA-426644?focusedCommentId=2191540) It looks like this a popular request from customers, so I modified Add App OIDC Api to have profile attributes, also included Assign Profile Attributes API (Update Apps API) to let customers have the option to include profile attributes after app creation.

**Pull request merged**